scalafmt-tests/src/test/resources/test/DefaultIndentOperator.stat

```diff
@@ -0,0 +1,75 @@
+indentOperator = default
```

Comment:
Is this test necessary? The default indent operator is tested throughout the test suite for every infix operator.

Reply:
From my understanding, it has no effect (because the indentOperator is already default) but it is still parsed and if the `"default"` wasn't part of the `ConfDecoder` it would fail:

```
→ java -jar scalafmt.jar -f /tmp/test.scala
StyleMap:44 [e]:
Type mismatch;
  found    : String (value: "bob")
  expected : IndentOperator
```

Reply:
This is the only reason I've added the test, to show the config `indentOperator = default` is working.

Should I remove it?

Reply:
Or maybe it is just not the right place to test for config parsing... I'll have another look to see where this test would make more sense

Reply:
The pattern match is simple enough so I think it's fine to skip testing the config parsing.

Reply:
Aight, I removed it